bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -367,6 +368,23 @@ ldap_add_custom_schema() {
     ldap_start_bg
 }
 
+########################
+# Add custom schemas
+# Globals:
+#   LDAP_*
+# Arguments:
+#   None
+# Returns
+#   None
+#########################
+ldap_add_custom_schemas() {
+    info "Adding custom Schemas : $LDAP_CUSTOM_SCHEMA_DIR ..."
+    find "$LDAP_CUSTOM_SCHEMA_DIR" -maxdepth 1 \( -type f -o -type l \) -iname '*.ldif' -print0 | sort -z | xargs --null -I{} bash -c ". /opt/bitnami/scripts/libos.sh && debug_execute slapadd -F \"$LDAP_ONLINE_CONF_DIR\" -n 0 -l {}"    
+    ldap_stop
+     while is_ldap_running; do sleep 1; done
```

Comment:
Can you fix the indentation, please?

Reply:
Can you elaborate a bit please? The indentation seems consistent with the surrounding code.

Reply:
Line 384 should have one less space to have a correct format.

Reply:
All set.